platform/jewel/samples/standalone/src/main/kotlin/org/jetbrains/jewel/samples/standalone/Main.kt

```diff
@@ -77,8 +86,10 @@ public fun main() {
                     processKeyShortcuts(keyEvent = keyEvent, onNavigateTo = MainViewModel::onNavigateTo)
                 },
                 content = {
-                    TitleBarView()
-                    ProvideMarkdownStyling { currentView.content() }
+                    CompositionLocalProvider(LocalPopupRenderer provides JDialogRenderer(window)) {
```

Comment:
I'm open to ideas on how to provide the window. Compose already has access to it, so they don't need to provide. But I haven't found any local provider for it

Reply:
We should probably do the same we do in the bridge with `JewelComposePanelWrapper.kt`